app/javascript/dashboard/routes/dashboard/settings/inbox/components/BotConfiguration.vue

```diff
@@ -86,6 +99,26 @@ export default {
         this.showAlert(this.$t('AGENT_BOTS.BOT_CONFIGURATION.ERROR_MESSAGE'));
       }
     },
+    async disconnectBot() {
+      try {
+        await this.$store.dispatch('agentBots/disconnectBot', {
+          inboxId: this.inbox.id,
+        });
+        this.showAlert(
+          this.$t('AGENT_BOTS.BOT_CONFIGURATION.DISCONNECTED_SUCCESS_MESSAGE')
+        );
+      } catch (error) {
+        this.showAlert(
```

Comment:
Check if there any error message exists in the error payload.

Reply:
@muhsin-k Handled this, please have a look